dnf/cli/commands/repoquery.py

```diff
@@ -195,7 +192,12 @@ def set_argparser(parser):
                              help=_('show changelogs of the package'))
         outform.add_argument('--qf', "--queryformat", dest='queryformat',
                              default=QFORMAT_DEFAULT,
-                             help=_('format for displaying found packages'))
+                             help=_('format for showing found packages: '
+                                    '"%%{name} %%{version} ..". '
```

Comment:
Please use `...` instead of `..` and I'd use a comman instead of full stop (since it's a bit awkward as the strings here aren't sentences):
> "%%{name} %%{version} ...",

And then on the next line change the capital "U" to lowercase :slightly_smiling_face:

Reply:
Done.